src/providers/export/tensorFlowRecords/tensorFlowReader.ts

```diff
@@ -62,4 +63,28 @@ export class TFRecordsReader {
     public toArray(): object[] {
         return this.imageMessages.map((imageMessage) => imageMessage.toObject());
     }
+
+    /**
+     * @recordPos - Record Position
+     * @key - Feature Key
+     * @type - Feature Type
+     * @description - Get a Int64 | Float | String | Binary value
+     */
+    public getFeature(recordPos: number, key: string, type: FeatureType): string | number | Uint8Array {
+        // Guard.expression(recordPos, (num) => num >= 0 && num < this.imageMessages.length);
+        const message = this.imageMessages[recordPos];
+        const feature = message.getContext().getFeatureMap().get(key);
+
+        switch (type) {
+            case FeatureType.String:
+                return textDecode(feature.getBytesList().array[0][0]);
+            case FeatureType.Binary:
+                return feature.getBytesList().array[0][0];
+            case FeatureType.Int64:
+                return feature.getInt64List().array[0][0];
+            case FeatureType.Float:
```

Comment:
is this needed when it's going to flow through to default anyway?

Reply:
Good catch, the default it's not needed tks